packages/styles/src/core/containers/test/containers.spec.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+
+function image (filename) {
+  return path.resolve(__dirname, `img/${filename}`);
+}
+
+function html (filename) {
+  const filePath = path.resolve(__dirname, `${filename}`);
+  return `file://${filePath}`;
+}
+
+function spec (reference, screenshot, url, mobile) {
```

Comment:
Nice idea!  We could refactor all our tests and create some kind of helper so we write less boilerplate code